doc/main.dox

```diff
@@ -119,6 +119,43 @@ Thus the following example specifies all sensors for four combinations of visit
 plus all sensors for one raft of two other visits for calibrated lsstSim data:
 - `--id visit=54123^55523 raft=1,1^2,1 --id visit=623459^293423 raft=0,0`
 
+\subsection pipeBase_argumentParser_rerun The --rerun Option
+
+The `--rerun` option is a convenient shortcut for specifying the output directory of a command line task and
+serves the same purpose as the `--output` argument. However, the value supplied to `--rerun` is relative to
+the "rerun" directory of the root input data repository (i.e. follow the "_parent" chain all the way back,
+then add "rerun" to that). `--rerun` saves the user from typing in the input repository twice. For example:
+
+`processCcd.py $root/hsc --output $root/hsc/rerun/vpk24/DM-4443`
+
+which will run processesCcd on the (HSC) data in "$root/hsc" and put the output in 
+"$root/hsc/rerun/vpk24/DM-4443" can be shortened by using `--rerun` to
+
+`processCcd.py $root/hsc ----rerun vpk24/DM-4443`
+
+This is possible because as of the adoption of RFC-95, outputs from the pipeline usually go within the 
```

Comment:
I suggest rewording this. RFC-95 is not useful information here (in my opinion, and if it is important, it should be a URL). I suggest this approach: "This is useful because outputs from pipelines usually go within...". If you want to provide more information you could include a link to RFC-95 later, but please first present what users need to know, then treat that link as "for more information see...".